gae-interop-testing/gae-jdk8/build.gradle

```diff
@@ -44,16 +44,15 @@ apply plugin: 'com.google.cloud.tools.appengine'  // App Engine tasks
 
 dependencies {
     providedCompile group: 'javax.servlet', name: 'servlet-api', version:'2.5'
-    compile 'com.google.appengine:appengine-api-1.0-sdk:1.9.59'
-    // Deps needed by all gRPC apps in GAE
-    compile libraries.google_api_protos
-    compile project(":grpc-okhttp")
-    compile project(":grpc-protobuf")
-    compile project(":grpc-stub")
-    compile (project(":grpc-interop-testing")) {
-        exclude group: "io.grpc", module: "grpc-netty-shaded"
+    runtimeOnly 'com.google.appengine:appengine-api-1.0-sdk:1.9.59'
+    implementation project(':grpc-netty')
+    implementation project(":grpc-stub")
+    implementation (project(':grpc-interop-testing')) {
+        exclude group: 'io.grpc', module: 'grpc-alts'
     }
-    compile libraries.netty_tcnative
+    implementation libraries.protobuf
+    implementation libraries.truth
```

Comment:
Why was this dep added?

Reply:
Trying to build without it:
```
/Users/zdapeng/git/grpc-java/gae-interop-testing/gae-jdk8/src/main/java/io/grpc/testing/integration/NettyClientInteropServlet.java:137: error: cannot find symbol
      assertTrue(builder instanceof NettyChannelBuilder);

```

The reason should be that `truth` was a dep of `grpc-interop-testing` which is now changed from compile to implementation. Only api/compile dep is transitive for compileClasspath across subproject. Implementation dep is transitive for runtimeClasspath.

Reply:
`assertTrue` is a junit method. This should probably be `implementation libraries.junit`.

Reply:
You are right. My mistake. Done.